controls/radrichtextbox/document-elements/table/formatting-tables.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Formatting Tables
+page_title: Formatting Tables
+description: Formatting Tables
+slug: radrichtextbox-formatting-tables
+tags: table
+published: True
+position: 3
+---
+
+# Formatting Tables 
+
+__RadRichTextBox__ exposes an API that allows you to use various methods to style the tables. The methods exposed by the API can be wired to a UI and get executed upon user interaction with this UI. To learn more about the __Formatting API__ of the __RadRichTextBox__, read [this topic]({%slug radrichtextbox-features-formatting-api%}).
+
+All methods that allow you to interact with Tables are listed here: [Tables Properties and Methods]({%slug radrichtextbox-features-api%}).
+
+
+## Formatting the Table in Code
+
+The bellow code shows how you can create a table in the code behind and set its styles:
+
+#### __[C#] Generate a table and set its borders and background__
```

Comment:
Add numbering to the caption:
__[C#] Example 1: Generate a table and set its borders and background__